### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by sjpadgett**
- fix and restyle code picker popup.
- dialog support 0 height auto sizing(mainly for alerts and confirm)
also fix for scroll. fingers crossed.

Fixes #

#### Short description of what this resolves:
Fix dialog scroll issue. Hopefully
Fix and restyle code picker.

### CODE REVIEW

**Comment by sjpadgett** [comment]
bringing in after travis so folks can test dialog. Tested scrolls on edge, chrome and ff. I think I may have figured out how to fix the scrolls on our tabs also...

File: interface/patient_file/encounter/find_code_popup.php [hunk 1]
```diff
@@ -49,200 +49,169 @@
...
+                    }
+                    echo ">" . xlt("Product") . "</option>\n";
+                    echo "</select>\n";
+                }
+                ?>
+                <div class="input-group mt-1">
+                    <input type='text' class='form-control' name='search_term' id="searchTerm"
+                        value='<?php echo attr($_REQUEST['search_term']); ?>'
+                        title='<?php echo xla('Any part of the desired code or its description'); ?>'
+                        placeholder="<?php echo xlt('Search for'); ?>" />
```
Comment (bradymiller) [inline interface/patient_file/encounter/find_code_popup.php:150]: xla
(finally, i get to point out a escaping thing :) )

**Comment by bradymiller** [comment]
my issue is minor, definitely bring this in after travis to keep momentum going and i'll so a quick commit later down the road on that.

File: interface/patient_file/encounter/find_code_popup.php (hunk 1 above)
Comment (sjpadgett) [reply interface/patient_file/encounter/find_code_popup.php:150]: you know me, I rarely pay a lot of attention to that...

**Comment by sjpadgett** [comment]
cool thanks. I was going to bed.

**Pull request merged**